test/bdd/pkg/cli/cli_steps.go

```diff
@@ -90,6 +92,27 @@ func (e *Steps) checkCreatedDID() error {
 	return nil
 }
 
+func (e *Steps) checkRecoveredDID() error {
+	const numberOfPublicKeys = 2
+
+	const numberOfServices = 2
+
+	doc, err := e.resolveDID(e.createdDID.ID)
+	if err != nil {
+		return err
+	}
+
+	if len(doc.PublicKey) != numberOfPublicKeys {
+		return fmt.Errorf("did doc public key is not equal to 2")
```

Comment:
you could use constant from above in your error msg

Reply:
done